src/components/RelatedMeetups.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { useStaticQuery, graphql } from 'gatsby';
+import Img from "gatsby-image"
+
+const RelatedMeetups = () => {
+	const data = useStaticQuery(graphql`
+		query {
+			angular: file(relativePath: { eq: "sisters/angularjs.png" }) {
+				childImageSharp {
+					fixed(height: 90, width: 90) {
+						...GatsbyImageSharpFixed
+					}
+				}
+			},
+			lunchJs: file(relativePath: { eq: "sisters/lunchjs.jpg" }) {
+				childImageSharp {
+					fixed(height: 90, width: 90) {
+						...GatsbyImageSharpFixed
+					}
+				}
+			},
+			nodeJs: file(relativePath: { eq: "sisters/nodejs.png" }) {
+				childImageSharp {
+					fixed(height: 90, width: 70) {
+						...GatsbyImageSharpFixed
+					}
+				}
+			},
+			reactJs: file(relativePath: { eq: "sisters/reactjs.png" }) {
+				childImageSharp {
+					fixed(height: 90, width: 90) {
+						...GatsbyImageSharpFixed
+					}
+				}
+			},
+			emberJs: file(relativePath: { eq: "sisters/emberjs.png" }) {
+				childImageSharp {
+					fixed(height: 90, width: 60) {
+						...GatsbyImageSharpFixed
+					}
+				}
+			},
+			ionicUtah: file(relativePath: { eq: "sisters/ionic.jpg" }) {
+				childImageSharp {
+					fixed(height: 90, width: 90) {
+						...GatsbyImageSharpFixed
+					}
+				}
+			},
+			vueJs: file(relativePath: { eq: "sisters/vuejs.jpg" }) {
+				childImageSharp {
+					fixed(height: 90, width: 90) {
+						...GatsbyImageSharpFixed
+					}
+				}
+			},
+			downtownCoding: file(relativePath: { eq: "sisters/downtown-coding.jpg" }) {
+				childImageSharp {
+					fixed(height: 58, width: 90) {
+						...GatsbyImageSharpFixed
+					}
+				}
+			}
+		}
+	`)
+
+	return (
+		<section>
+			<h2 id="Meetups">Other Web Technology Meetups in Utah</h2>
+			<div className="sisters">
```

Comment:
You mentioned during the video review possibly using a new component for this. Great idea I think. If we update the query to use allImageSharp/allChildIamge... etc then we can map overall images and create a sisters component.